pingdom/resource_pingdom_check.go

```diff
@@ -545,6 +558,12 @@ func resourcePingdomCheckRead(ctx context.Context, d *schema.ResourceData, meta
 		}
 	}
 
+	if ck.Status == "ipv6" {
```

Comment:
? i think this is wrong. there might be gap in the golang client, ill check

Reply:
You are right, that check was wrong.